lib/hammer_cli_katello.rb

```diff
@@ -43,9 +43,19 @@ def self.exception_handler_class
                                           'hammer_cli_katello/organization'
                                          )
 
-  HammerCLI::MainCommand.lazy_subcommand("gpg", _("Manipulate GPG Key actions on the server"),
+  HammerCLI::MainCommand.lazy_subcommand("gpg",
+                                         _("Manipulate GPG Key actions on the server"),
                                          'HammerCLIKatello::GpgKeyCommand',
-                                         'hammer_cli_katello/gpg_key'
+                                         'hammer_cli_katello/gpg_key',
+                                         :warning =>
```

Comment:
There's actually a `deprecated` key you can pass here, like this: https://github.com/Katello/hammer-cli-katello/blob/d70bd7ad6d02ba2b9e9d0d67b0d5e51109a4b760/lib/hammer_cli_katello/command_extensions/lifecycle_environments.rb#L8